refl1d/webview/client/src/standalone_signal.ts

```diff
@@ -1,42 +1,42 @@
 export class Signal {
-    name: string;
```

Comment:
It's worth double-checking, but I don't think this code is used anywhere else in this repo anymore, and we might be able to remove it.

Reply:
i believe you are correct!

Reply:
It got moved over here: https://github.com/reflectometry/refl1d-demo